client/attest.go

```diff
@@ -140,6 +142,12 @@ type SevSnpDevice struct {
 	Device sg.Device
 }
 
+// TdxDevice encapsulates the TDX attestation device to add its attestation quote
```

Comment:
I don't think you need a wrapper type if you're not adding extra information. Your functions can just take a Device argument. Then you don't need to have Open/Close wrapper functions.

Reply:
I tried to keep the code uniform in the file, similar to SevSnpDevice struct, I have created TdxDevice struct. Should I remove it?

Reply:
Nah, leave it in I suppose.

Reply:
Okay.